### PR TITLE
Story area mission area styles

### LINKED ISSUE #33: Testing For Story Area
As a dev I want to make sure my Story Area functions how I want it too

### LINKED ISSUE #34: Mission Area testing
As a dev I want to make sure my Mission Area component functions how I want it too.

### PR DESCRIPTION
**Pull request by taylorjohnson141**
### What does this PR do?
Adds Basic styling and tests to Story Area and Mission Area

### Where should the reviewer start?
StoryArea.js

### How should this be manually tested?
npm test
### Any background context you want to provide?

Will add more styles and possibly more tests in the future

### What are the relevant tickets?
closes #33 
closes #34

### CODE REVIEW

File: src/components/StoryArea/Story.test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    numbers2: "4",
+   'occupation-name': "Accountant",
+   'plural-animal-name': "Dogs",
+  'restaurant-name': "Romanos",
+   'singular-animal-name': "Cat",
+    'sport-name': "Football",
+    'zoo-animal': "Monkey"}
+
+    render(<StoryArea currentIdentity = {currentIdentity}/>)
+    let IdArticle = screen.getByText('Your name is Glinda Hamilton. You work for Sears as a/an Accountant. You have 3 children and your spouse’s name is Gertrude. The two of you met at Romanos. You have a Cat and two Dogs In your free time, you like to coach a youth Football team and play Catan on weekends with your social club. You enjoy Punk novels and watching Pop movies. You have 4 close friends who often call you by your nickname, the Orange Monkey.')
```
Comment (timkeresey) [inline src/components/StoryArea/Story.test.js:31]: We probably want this naming to be in lowercase.

File: src/components/MissionArea/MissionArea.js [hunk 2]
```diff
@@ -3,20 +3,33 @@ import './MissionArea.css';
...
+  let languages;
+  if(!currentCountry.regionalBlocs[0]){
+    takeOver = currentCountry.region
+  }else{
+    takeOver = `the ${currentCountry.regionalBlocs[0].name}`
+  }
+  if(!currentCountry.languages[1]){
+    languages = currentCountry.languages[0].name
+  } else{
+    languages = `${currentCountry.languages[0].name} and ${currentCountry.languages[1].name}`
```
Comment (ericberglund117) [inline src/components/MissionArea/MissionArea.js:16]: This conditional rendering looks good. It takes care of the small details in our mission area

**Review by timkeresey — COMMENTED** [review]
thanks for making these updates

File: src/components/StoryArea/Story.test.js (hunk 1 above)
Comment (ericberglund117) [inline src/components/StoryArea/Story.test.js:31]: Great job on the tests, we might need to break line 31 down so it's not one big line for linting purposes

**Review by ericberglund117 — APPROVED** [review]

**Pull request merged**